app/src/main/AndroidManifest.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
These changes shouldn't have been included in this PR. Just makes noise

Reply:
Not sure why these are even here. I never touched these files but for some reason they seem to think they were changed.